### ISSUE TITLE
Can't reproduce the outputs from the detect.py file on images/bus or images/zidane locally

### ISSUE THREAD

**Issue by RobinBram** [ultralytics/yolov5#2080]
Hi!

Having real trouble with this repository. Have tried training on custom data, everyhing seems fine but the model does not do any progress. I can't either get the detect.py to detect any objects in the image when following the code from the Tutorial at https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb#scrollTo=zR9ZbuQCH7FX . Here are some outputs from my notebook: 
Setup complete. Using torch 1.7.0 _CudaDeviceProperties(name='GeForce GTX 1650', major=7, minor=5, total_memory=4096MB, multi_processor_count=14)

And when running !python detect.py --weights yolov5l.pt --img 640 --conf 0.25 --source data/images/

Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='data/images/', update=False, view_img=False, weights=['yolov5l.pt'])
Downloading https://github.com/ultralytics/yolov5/releases/download/v4.0/yolov5l.pt to yolov5l.pt...

Fusing layers... 
image 1/2 C:\Users\robr\Documents\Python Scripts\yolov5\data\images\bus.jpg: 640x480 Done. (0.227s)
image 2/2 C:\Users\robr\Documents\Python Scripts\yolov5\data\images\zidane.jpg: 384x640 Done. (0.158s)
Results saved to runs\detect\exp6
Done. (0.454s)
YOLOv5 v4.0-53-g2acbe96 torch 1.7.0 CUDA:0 (GeForce GTX 1650, 4096.0MB)

  0%|          | 0.00/90.2M [00:00<?, ?B/s]
  2%|1         | 1.44M/90.2M [00:00<00:06, 15.0MB/s]
  5%|5         | 4.62M/90.2M [00:00<00:03, 25.8MB/s]
  9%|9         | 8.52M/90.2M [00:00<00:02, 32.6MB/s]
 16%|#5        | 14.4M/90.2M [00:00<00:01, 43.1MB/s]
 27%|##6       | 24.3M/90.2M [00:00<00:01, 62.8MB/s]
 37%|###6      | 33.1M/90.2M [00:00<00:00, 72.4MB/s]
 44%|####4     | 40.0M/90.2M [00:00<00:00, 68.1MB/s]
 52%|#####1    | 46.5M/90.2M [00:00<00:00, 62.3MB/s]
 58%|#####8    | 52.5M/90.2M [00:00<00:00, 61.4MB/s]
 65%|######4   | 58.4M/90.2M [00:01<00:00, 61.5MB/s]
 71%|#######1  | 64.3M/90.2M [00:01<00:00, 56.4MB/s]
 77%|#######7  | 69.8M/90.2M [00:01<00:00, 56.9MB/s]
 84%|########3 | 75.5M/90.2M [00:01<00:00, 57.1MB/s]
 90%|########9 | 81.1M/90.2M [00:01<00:00, 57.3MB/s]
 96%|#########5| 86.6M/90.2M [00:01<00:00, 57.4MB/s]
100%|##########| 90.2M/90.2M [00:01<00:00, 56.4MB/s]
Model Summary: 392 layers, 47025981 parameters, 0 gradients, 115.4 GFLOPS.

I have tried all the different pretrained versions of yolo. Is this a version issue in some way and what can I do to get this to work?

**Comment by RobinBram** [comment]
When creating a new environment and installing pytorch for cpu, it works fine!

**Comment by glenn-jocher** [comment]
May be a windows or environment issue. I'll paste our default environmental issue below:

It appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual Python 3.8 environment, clone the latest repo (**code changes daily**), and `pip install -r requirements.txt` again. We also highly recommend using one of our verified environments below. 


## Requirements

**Python 3.8** or later with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) dependencies installed, including **`torch>=1.7`**. To install run:
```bash
$ pip install -r requirements.txt
```


## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab Notebook** with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
- **Kaggle Notebook** with free GPU: [https://www.kaggle.com/models/ultralytics/yolov5](https://www.kaggle.com/models/ultralytics/yolov5)
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) 
- **Docker Image** https://hub.docker.com/r/ultralytics/yolov5. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) ![Docker Pulls](https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker)


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are passing. These tests evaluate proper operation of basic YOLOv5 functionality, including training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), testing ([test.py](https://github.com/ultralytics/yolov5/blob/master/test.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/models/export.py)) on MacOS, Windows, and Ubuntu.

**Comment by cloudwsx** [comment]
maybe it is a CUDA problem.
https://github.com/ultralytics/yolov5/issues/1910

**Comment by RobinBram** [comment]
It works now. Degraded CUDA to 10.1 and reinstalled everything

**Comment by RobinBram** [comment]
![image](https://user-images.githubusercontent.com/70885556/106437678-cab8ec00-6475-11eb-9e63-b56c6ee9b707.png)
Here are some training runs I did with the "bug".

**Comment by devpawann** [comment]
> Degraded CUDA to 10.1

How do you exactly do that?
It's not working for me
